.github/workflows/run-integration-tests.yml

```diff
@@ -0,0 +1,104 @@
+name: Run Integration Tests
```

Comment:
I assume this file is due to merge conflict resolution. Could you please remove this file? Then I am happy to approve this PR.